fixtures/plugins/astro/knip.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+  ignore: ['**/*.d.ts'],
+  compilers: {
+    astro: (text: string) => [...text.matchAll(/import[^;]+/g)].join('\n'),
+    css: (text: string) => [...text.matchAll(/(?<=@)import[^;]+/g)].join('\n'),
```

Comment:
I think we also need a `mdx` compiler here (identical to `astro` to extract the `import` statements?)

Reply:
[`05e9895` (#298)](https://github.com/webpro/knip/pull/298/commits/05e9895c49f7d373caa4122b6d846f6235188860)